docs/package.json

```diff
@@ -10,6 +10,8 @@
     "update-contributors": "node scripts/update-contributors.js"
   },
   "devDependencies": {
+    "@shikijs/vitepress-twoslash": "^1.22.2",
+    "openapi-metadata": "workspace:*",
```

Comment:
For some reason, the Cloudflare preview isn’t showing up, but adding this local dependency will break the docs publishing, because we need to build this.

To fix this, change the following in the root `package.json`:

```diff
-   "build": "pnpm run -r --filter \"!*docs\" --aggregate-output build",
+   "build": "pnpm run -r --aggregate-output build",
```

The great part about pnpm is it’s smart enough to figure out the correct order in which to build everything. But it’s getting to the point where we probably need a more comprehensive solution (#2016).

We’ll need this change to merge it. But once this is in, everything else looks good!

Reply:
Not relevant anymore since you implemented turbo :rocket: